packages/core/src/checkout/checkout-action-creator.spec.ts

```diff
@@ -151,15 +153,15 @@ describe('CheckoutActionCreator', () => {
             );
         });
 
-        it('calls loadConfig in parallel', () => {
+        it('does not call loadCheckout until loadConfig resolves', () => {
```

Comment:
as discussed with @animesh1987, after changed the loadCheckout, the loading is in sequence, like loadDefaultCheckout, so have to change this test.